granulate_utils/__init__.py

```diff
@@ -2,4 +2,13 @@
 # Copyright (c) Granulate. All rights reserved.
 # Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
 #
+import sys
+from functools import lru_cache
+
 __version__ = "0.0.1"
+WINDOWS_PLATFORM_NAME = "win32"
+
+@lru_cache(maxsize=None)
```

Comment:
It's okay to add it here, but we can also just add in gProfiler repo. This granulate-utils repo is for utilities shared between gProfiler and other projects, and currently `is_windows` is required only by gProfiler so we can place it there.

Reply:
If I move it from here, there's no way for __init__.py in gprofiler/profilers to check what modules to import. Except you are fine with me defining it in gprofiler/profilers/__init__.py before the import statements. I thought it would be cleaner to have it in your utils package.

Reply:
It can go to `gprofiler/utils/__init__.py` or possibly to `gprofiler/__init__.py` if you need it very very early in the imports flow